rust/libnewsboat/src/utils.rs

```diff
@@ -445,6 +445,24 @@ pub fn make_title(rs_str: String) -> String {
     }
 }
 
+/// Get the current working directory.
+/// Returns an empty string on error
+pub fn getcwd() -> String {
+    std::env::current_dir()
+        .and_then(|path| {
+            path.to_str()
+                .ok_or(std::io::Error::new(
+                    std::io::ErrorKind::Other,
+                    "Invalid unicode in path",
```

Comment:
When returning a `PathBuf` we also wouldn't need to error out if it doesn't contain unicode.